dev-support/release-vm/README.md

```diff
@@ -0,0 +1,141 @@
+<!---
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# HBase Release Env
+
+This is a vagrant project that provides a virtual machine environment suitable
+for running an Apache HBase release.
+
+Requires:
+* [VirtualBox](http://virtualbox.org)
+* [Vagrant](http://virtualbox.org)
+* The private portion of your signing key avilable in the local GPG agent
+* The private portion of your Github authentication key available in either the local GPG agent or
+  local SSH agent
+
+## Usage
+
+Unlock the local keyring before proceeding (this should prompt you for your GPG passphrase). For
+example, assuming you have an authentication key configured in your keyring, this will do the
+trick.
+
+All terminal commands used below are assumed to be run with the current working directory as the
+location containing the `Vagrantfile`.
+
+The term "Host" is used to mean the environment that runs the Vagrant process. The term "Guest" is
+used to mean the virtual machine managed by the Host.
+
+### Ensure credentials work from the Host OS
+
+The ssh- and gpg-agent forwarding configuration used here assumes that your credentials work
+on the Host. Verify both are working before you proceed with the Guest. Additionally, using the
+credentials requires you to unlock the respective keyring, the state of which is persisted by the
+agent process or processes running on the Host.
+
+See instructions in [`create-release`](../create-release/README.txt) regarding proper
+configuration of ssh- and gpg-agents.
+
+Assuming the git repo origin is on GitHub, the following command will ensure that your ssh
+credentials are working. On the Host, run:
+
+```sh
+host:~$ ssh -T git@github.com
+Hi <you>! You've successfully authenticated, but GitHub does not provide shell access.
+```
+
+Likewise, ensure you have an encryption key that can be used to sign a file. Again, on the Host,
+run:
```

Comment:
No cd into release-vm dir?

Reply:
No, not down here. See the second paragraph added up above under the "Usage" heading. I wrote it that way so that if this readme gets moved around later, the anchor of reference holds true.